frontend/src/i18n/locales/features/admin-form/sidebar/fields/en-sg.ts

```diff
@@ -0,0 +1,194 @@
+import { Fields } from '.'
+
+export const enSG: Fields = {
+  builder: {
+    title: 'Fields',
+    createField: 'Create field',
+    image: 'Image',
+    statement: 'Paragraph',
+    section: 'Heading',
+    attachment: 'Attachment',
+    checkbox: 'Checkbox',
+    date: 'Date',
+    decimal: 'Decimal',
+    dropdown: 'Dropdown',
+    countryRegion: 'Country/Region',
+    email: 'Email',
+    homeNumber: 'Home number',
+    mobileNumber: 'Mobile number',
+    longText: 'Long answer',
+    nik: 'NIK / KK',
+    number: 'Number',
+    radio: 'Radio',
+    rating: 'Rating',
+    shortAnswer: 'Short answer',
+    table: 'Table',
+    yesNo: 'Yes/No',
+    children: 'Children',
+  },
+  commonFieldComponents: {
+    title: 'Question',
+    description: 'Description',
+    required: 'Required',
+    noCharactersAllowed: 'Number of characters allowed',
+    charactersAllowedPlaceholder: 'Number of characters',
+  },
+  radio: {
+    others: 'Others',
+    options: {
+      title: 'Options',
+      placeholder: 'Enter one option per line',
+    },
+    duplicateOptionsError: 'Please remove duplicate options',
+    otherInvalidInputError: 'Harap masukan input lainnya',
```

Comment:
This isn't English at all. Please find the corresponding English in the codebase by comparison and inspection

Reply:
Thanks for the catch. Made the amendments.

Reply:
Have the amendments been made? The text still reflects a language other than English

Reply:
Just pushed the amendments.

Reply:
This isn't an amendment at all; you removed the translation altogether. Could you please find the original English text in FormSG? You can do this by identifying the component where this key is used in the fork you were briefed on, and then locating the original text in FormSG.

Reply:
Apologies, I misunderstood the task before. 

Just added the respective translation, it was found in the `RadioField.tsx` file. Do let me know if its good now.